src/tribol/search/InterfacePairFinder.cpp

```diff
@@ -210,7 +210,7 @@ class CartesianProduct : public SearchBase {
     ContactMode cmode = m_coupling_scheme->getContactMode();
 
     bool auto_contact_check = m_coupling_scheme->getParameters().auto_contact_check;
-    auto binning_proximity_scale = m_coupling_scheme->getParameters().binning_proximity_scale;
+    auto binning_proximity_scale = m_coupling_scheme->getEffectiveBinningProximityScale();
```

Comment:
I think this is ok because it is a local variable inside the grid binning, but potentially confusing. I first looked at this thinking that we should be setting `auto binning_proximity_scale` to the binning proximity scale stored on the coupling scheme, but recalled that we always want to be using the effective binning scale. Just as long as it is clear that we actually want to be using the effective.

Reply:
Right, I can see why this might be confusing.  I added a note to the `getEffectiveBinningProximityScale()` that should clear up its usage.